### PR TITLE
Ghost sign in

### PR DESCRIPTION
**Pull request by MillerAdulu**
## Description

Our mobile app only has social sign-in with Google. Google Play requires that you submit working credentials with the app to log in and review the app. Google requires every account to have 2FA setup, so it's unrealistic to hand them your personal account because the 2FA will fail. So this PR is to seed a dummy user on the app which the app can then use in Ghost mode to have the auth be circumvented on the app by long pressing the Fluttercon logo. This way they can log in to review the app and then once the app has been released, on the mobile end, we shall disable review mode so that normal users don't use this as a feature to bypass the login.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [x] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Test Changes on Device
You can download your APK from Firebase App Distribution once this PR builds successfully via the following link: https://appdistribution.firebase.dev/i/c796669942f8a811

### CODE REVIEW

**Comment by MillerAdulu** [comment]
@NewtonMutugi,

Please don't merge yet when you are done looking. 

Needs https://github.com/droidconKE/droidconEventsBanckend/pull/41

**Comment by NewtonMutugi** [comment]
Okay let me have a look

File: lib/features/auth/ui/sign_in.dart [hunk 1]
```diff
@@ -53,7 +56,38 @@ class SignInScreen extends StatelessWidget {
                   mainAxisAlignment: MainAxisAlignment.center,
                   children: [
                     const Spacer(),
-                    const Image(image: AssetImage(AppAssets.flutterConKeLogo)),
+                    GestureDetector(
+                      onLongPress: () {
+                        final config = getIt<FirebaseRepository>().getConfig();
+                        if (config.isInReview &&
+                            config.appVersion == Misc.getAppVersion()) {
```
Comment (mboyamike) [inline lib/features/auth/ui/sign_in.dart:63]: Does this mean config.appversion will be the version of the app that's in review?

File: lib/main_production.dart [hunk 2]
```diff
@@ -14,8 +14,8 @@ Future<void> main() async {
   });
   FlutterConConfig(
     values: FlutterConValues(
-      baseDomain: 'api.droidcon.co.ke',
-      urlScheme: 'https',
+      baseDomain: '127.0.0.1:8000',
```
Comment (mboyamike) [inline lib/main_production.dart]: For prod, won't this throw an error when making requests?

Comment (mboyamike) [reply lib/main_production.dart]: Oh never mind. Seen you fixed this

**Review by NewtonMutugi — APPROVED** [review]
LGTM

File: lib/features/auth/ui/sign_in.dart (hunk 1 above)
Comment (MillerAdulu) [reply lib/features/auth/ui/sign_in.dart:63]: Yes yes. That way there's no possibility of a leak with backward versions. 

The config to control this as well is on firebase remote config.

**Review by mboyamike — APPROVED** [review]

**Review by mboyamike — APPROVED** [review]

**Pull request merged**